samples/system-test/spanner.test.js

```diff
@@ -41,624 +41,705 @@ const spanner = new Spanner({
-  t.regex(output, /Waiting for operation to complete\.\.\./);
-  t.regex(output, /Added the MarketingBudget column\./);
-});
+    assert.strictEqual(
+      new RegExp(/SingerId: 1, AlbumId: 1, AlbumTitle: Total Junk/).test(
+        output
+      ),
+      true
+    );
+  });
 
-// update_data
-test.serial(`should update existing rows in an example table`, async t => {
-  const results = await tools.runAsyncWithIO(
-    `${crudCmd} update ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`,
-    cwd
-  );
-  const output = results.stdout + results.stderr;
-  t.regex(output, /Updated data\./);
-});
+  // read_data
+  it(`should read an example table`, async () => {
+    const results = await tools.runAsyncWithIO(
+      `${crudCmd} read ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`,
+      cwd
+    );
+    const output = results.stdout + results.stderr;
+    assert.strictEqual(
+      new RegExp(/SingerId: 1, AlbumId: 1, AlbumTitle: Total Junk/).test(
+        output
+      ),
+      true
+    );
+  });
 
-// read_stale_data
-test.serial(`should read stale data from an example table`, t => {
-  t.plan(2);
-  // read-stale-data reads data that is exactly 15 seconds old.  So, make sure
-  // 15 seconds have elapsed since the update_data test.
-  return new Promise(resolve => setTimeout(resolve, 16000)).then(async () => {
+  // add_column
+  it(`should add a column to a table`, async () => {
     const results = await tools.runAsyncWithIO(
-      `${crudCmd} read-stale ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`,
+      `${schemaCmd} addColumn ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`,
       cwd
     );
     const output = results.stdout + results.stderr;
-    t.regex(
-      output,
-      /SingerId: 1, AlbumId: 1, AlbumTitle: Total Junk, MarketingBudget: 100000/
+    assert.strictEqual(
+      new RegExp(/Waiting for operation to complete\.\.\./).test(output),
+      true
     );
-    t.regex(
-      output,
-      /SingerId: 2, AlbumId: 2, AlbumTitle: Forever Hold your Peace, MarketingBudget: 500000/
+    assert.strictEqual(
+      new RegExp(/Added the MarketingBudget column\./).test(output),
+      true
     );
   });
-});
 
-// query_data_with_new_column
-test.serial(
-  `should query an example table with an additional column and return matching rows`,
-  async t => {
+  // update_data
+  it(`should update existing rows in an example table`, async () => {
+    const results = await tools.runAsyncWithIO(
+      `${crudCmd} update ${INSTANCE_ID} ${DATABASE_ID} ${PROJECT_ID}`,
+      cwd
+    );
+    const output = results.stdout + results.stderr;
+    assert.strictEqual(new RegExp(/Updated data\./).test(output), true);
+  });
+
+  // read_stale_data
+  it(`should read stale data from an example table`, () => {
+    // read-stale-data reads data that is exactly 15 seconds old.  So, make sure
+    // 15 seconds have elapsed since the update_data test.
+    return new Promise(resolve => setTimeout(resolve, 16000)).then(async () => {
```

Comment:
Make this an async function, and use the `await new Promise(r => setTimeout...` trick :)

Reply:
Thank you Justin for your feedback. I did the change.